charts/aws-efs-csi-driver/templates/csidriver.yaml

```diff
@@ -8,3 +8,10 @@ metadata:
     "helm.sh/resource-policy": keep
 spec:
   attachRequired: false
+  {{- if .Values.iamAuth }}
```

Comment:
maybe something  a bit more descriptive for the helm value / volumeAttribute? `podIAMAuth`? `podIAMAuthorization? since this is b asically "mounting using the IAM authorization using the pod's IAM credentials" https://docs.aws.amazon.com/efs/latest/ug/mounting-IAM-option.html

Reply:
FIxed, used `podIAMAuthorization` as it carry more meaning.